packages/components/src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { useStyles } from "../../hooks/useStyles";
+import { Color, EDSStyleSheet } from "../../styling";
+import { View } from "react-native";
+import { Typography } from "../Typography";
+import { PressableHighlight } from "../PressableHighlight";
+import { Icon, IconName } from "../Icon";
+import { Button } from "../Button";
+
+export type ChipProps = {
+    /**
+     * Text to render in the chip.
+     */
+    title: string;
+    /**
+     * The chip variant. Determines the visual expression of the chip.
+     */
+    variant?: "default" | "error" | "active";
+    /**
+     * Whether or not the chip should respond to touch events and render its disabled state.
+     */
+    disabled?: boolean;
+    /**
+     * Callback method invoked when the user presses the chip. If not defined, the chip is not pressable.
+     */
+    onPress?: () => void;
```

Comment:
Its more clear the way it is now, but the disable prop is not really necessary here as onPress={undefined} disables it.

Reply:
ah yeah I see, only the pressable is disabled